manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -283,6 +286,28 @@ func init() {
 	SchemeBuilder.Register(&ManageIQ{}, &ManageIQList{})
 }
 
+func fetchIntrospectionUrl(providerUrl string) (introspectionUrl string, errMsg string) {
```

Comment:
This doesn't belong in the `v1alpha1` package. I think this all should go at least into the controller, possibly into the httpd component.